src/components/icons/chromicons.js

```diff
@@ -1,52 +1,114 @@
+import React from 'react';
+import Link from 'next/link';
+
 export const Chromicons = ({ ...rootProps }) => (
-  <svg width={172} height={24} viewBox="0 0 172 24" fill="none" {...rootProps}>
-    <path
-      d="M15.84 10.341a7.088 7.088 0 10-.187 9.826M21.663 8.059v14.188M33.96 8.059v14.188M21.663 15.153H33.96"
-      stroke="#fff"
-      strokeWidth={2}
-      strokeMiterlimit={10}
-      strokeLinecap="round"
-    />
-    <path
-      d="M40.11 22.247V8.096h7.114a4.44 4.44 0 014.439 4.391v0a4.439 4.439 0 01-4.44 4.484h-.264l5.45 5.272M88.667 22.247V8.096l-6.245 6.051-6.053-6.051v14.15M94.794 8.059v14.188"
-      stroke="#fff"
-      strokeWidth={2}
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-    <path
-      d="M113.292 10.36a7.086 7.086 0 00-10.999.739 7.085 7.085 0 003.024 10.601 7.09 7.09 0 007.788-1.513"
-      stroke="#fff"
-      strokeWidth={2}
-      strokeMiterlimit={10}
-      strokeLinecap="round"
-    />
-    <path
-      d="M124.918 22.247a7.094 7.094 0 100-14.188 7.094 7.094 0 000 14.188zM137.718 22.265V8.06l12.298 14.206V8.06M155.996 20.216a9.607 9.607 0 005.573 1.997c3.396 0 6.149-1.582 6.149-3.535 0-4.55-12.297-2.658-12.297-7.05 0-1.952 2.752-3.535 6.148-3.535a8.066 8.066 0 015.711 2.224"
-      stroke="#fff"
-      strokeWidth={2}
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-    <path
-      d="M56.75 2.918h14.187m-7.094 5.16a7.093 7.093 0 110 14.186 7.093 7.093 0 010-14.187v0z"
-      stroke="url(#prefix__paint0_linear)"
-      strokeWidth={2}
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-    <defs>
-      <linearGradient
-        id="prefix__paint0_linear"
-        x1={55.628}
-        y1={18.728}
-        x2={72.058}
-        y2={2.3}
-        gradientUnits="userSpaceOnUse"
+  <Link href="/" passHref>
+    <a className="cursor-pointer">
```

Comment:
For accessibility reasons, since this is a link with an SVG element that cannot be read by screenreader, I think adding this will do the trick:

```suggestion
    <a className="cursor-pointer" aria-label="Navigate home">
```